src/functions.js

```diff
@@ -36,40 +49,79 @@ function cities( capitals, formatter ){
   // such as 'Paris is the capital of France'
 
   // apply formatter to each object in capitals array and return an array of resulting sentences
+  var newArr = [];
+  for (i = 0; i < capitals.length; i++) {
+    newArr[i] = formatter(capitals[i]);
+  };
+  return newArr;
 }
 
-function largerThanTen( numbers ){
+function largerThanTen(numbers) {
   // numbers is an array of numbers
   // return a new array that contains only numbers from the input array which are greater than 10
   // hint: filter
+  var newArr = numbers.filter(function (item) {
+    return item > 10;
+  });
+  return newArr;
 }
 
-function even( numbers ){
+function even(numbers) {
   // numbers is an array of numbers
   // return a new array that contains only even numbers from the input array
   // hint: you may want to use the modulus operator '%'
+  var newArr = [];
+  for (i = 0; i < numbers.length; i++) {
+    if (numbers[i] % 2 == 0) {
+      newArr.push(numbers[i]);
+    } else { continue }
+  };
+  return newArr;
 }
```

Comment:
Try to avoid using `==` as it is capable of implicit type conversion and so may not give you the result you expect. Unless you have a very good reason, always use the strict equality operator, `===`